<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Connect</string>
    </property>
    <widget class="QTextEdit" name="textEditClientConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>591</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Chat</string>
    </property>
    <widget class="QTextEdit" name="textEditEnter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>490</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidgetChat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>461</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>All</string>
      </attribute>
      <widget class="QTextEdit" name="textEditChat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>541</width>
         <height>411</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Contacts</string>
    </property>
    <widget class="QListWidget" name="listWidgetContacts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonChat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Chat</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>330</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Online</string>
    </property>
    <widget class="QListWidget" name="listWidgetOnline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOnlineRefresh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOnlineRemove">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
